### PR TITLE
Proposal of small corrections and new challenge for lesson 01

### PR DESCRIPTION
**Pull request by joaquinmoris**
Three changes are proposed:
- Small typo correction on lesson 01
- New challenge on lesson 01 introducing participants to a more advanced use of read.table, complementary to the advanced lesson available
- More details on one of the tips of lesson 5

### CODE REVIEW

**Comment by gvwilson** [comment]
Looks useful to me - a few comments inline.

File: 01-starting-with-data.Rmd [hunk 1]
```diff
@@ -41,7 +41,7 @@ To do all that, we'll have to learn a little bit about programming.
 To load our inflammation data, first we need to tell our computer where is the file that contains the values. We have been told its name is `inflammation-01.csv`. This is very important in R, if we forget this step we’ll get an error message when trying to read the file. We can change the current working directory using the function `setwd`. For this example, we change the path to the directory we just created:
 
 ```{r,eval=FALSE}
-setwd("~/Desktop/r-novice-inflammation/)
+setwd("~/Desktop/r-novice-inflammation/")
```
Comment (gvwilson) [inline 01-starting-with-data.Rmd]: Hm - has this bug been in there the whole time?

File: 01-starting-with-data.Rmd [hunk 2]
```diff
@@ -64,6 +64,12 @@ The filename needs to be a character string (or [string](reference.html#string)
 > importing your own data in the future. You can learn more about these
 > options in this supplementary [lesson](01-supp-read-write-csv.html).
 
+> ## Challenge - Reading different decimal point formats {.challenge}
```
Comment (gvwilson) [inline 01-starting-with-data.Rmd]: Don't need the word "Challenge" (but we can take that out).

File: 01-starting-with-data.Rmd [hunk 3]
```diff
@@ -64,6 +64,12 @@ The filename needs to be a character string (or [string](reference.html#string)
...
+> Depending on the country you live in, your standard can use the dot or the comma as decimal mark.
+> Also, different devices or software can generate data with different decimal points. 
+> Take a look at '?read.csv' and write the code to load a file called commadec.txt that has numeric values with commas as decimal mark, separated by semicolons.
```
Comment (gvwilson) [inline 01-starting-with-data.Rmd]: Please use back-quotes around `?read.csv` to format as code, and back-quotes around `commadec.txt`.

File: 05-cmdline.Rmd [hunk 4]
```diff
@@ -70,7 +70,8 @@ Rscript session-info.R
 
 > ## Tip {.callout}
 >
-> If that did not work, remember that you must be in the correct directory.
+> If that did not work, you might have seen an error message indicating that the file session-info.R does not exist.
```
Comment (gvwilson) [inline 05-cmdline.Rmd]: Please use back quotes around the filename.

File: 01-starting-with-data.Rmd (hunk 1 above)
Comment (hdashnow) [inline 01-starting-with-data.Rmd]: This has been fixed by #184. It's also probably the source of the merge conflict, so I suggest removing it from this PR.

**Comment by hdashnow** [comment]
@joaquinmoris thanks for the PR. Are you happy to make those changes that @gvwilson suggested so I can merge? Let me know if you have any questions.

**Comment by hdashnow** [comment]
Rebased this in #206, thanks @joaquinmoris!

**Pull request closed without merging**